tests/motion/motion_comment.create.yml

```diff
@@ -0,0 +1,41 @@
+---
+action: motion_comment.create
+db:
+  motion_comment_section/2/read_group_ids: [1,2]
+  motion_comment_section/2/write_group_ids: [2,3]
+  motion_comment_section/2/meeting_id: 1
+
+payload:
+  section_id: 2
+
+cases:
+- name: User not in section
+  is_allowed: false
+
+- name: Motion manager
+  permission: motion.can_manage
+  is_allowed: false
```

Comment:
In OS3 a user with `motion.can_manage` is allowed to see and modify every comment and comment section (Watch out for the `exact` flag in the OS3 code: https://github.com/OpenSlides/OpenSlides/blob/master/server/openslides/utils/auth.py#L115)

Reply:
This is true for all motion comment actions; I won't note it everywhere

Reply:
I asked @normanjaeckel this question and he told me, that a manager could not do the actions in os3. I have to preference. If you both know what you want, then I can change it.

Reply:
Right, I switched comments and comment sections in my head - but the meeting admin is allowed to do it. Please add a test, that the meeting admin (regardless if he is in any of the the access groups) can modify comments.

Reply:
done